docs/usage.rst

```diff
@@ -351,12 +351,7 @@ For example, the `RSVP.js library
 
 `Bluebird
 <http://bluebirdjs.com/>`_ and other promise libraries report unhandled rejections to a global DOM event, ``unhandledrejection``:
-
-.. code-block:: javascript
-
-    window.onunhandledrejection = function(evt) {
-        Raven.captureException(evt.reason);
-    };
+In this case, you don't need to do anything, we already got you covered.
```

Comment:
maybe mention `captureUnhandledRejections` here?

Reply:
small nit, i would say "we've" not "we"

Reply:
Updated. And added this note.